lib/identity/config.rb

```diff
@@ -2,6 +2,10 @@ module Identity
   module Config
     extend self
 
+    def finalize_shared_secret
```

Comment:
Hey, this file sort of fell into disrepair, but it's still mostly alphabetized. Mind just sorting this between "c" and "h"? Thanks!